src/weakptr.c

```diff
@@ -103,24 +101,43 @@ static inline void FORGET_WP(Obj wp, UInt pos)
 
 #define STORED_LEN_WPOBJ(wp)                 ((Int)(CONST_ADDR_OBJ(wp)[0]))
 
+
 /****************************************************************************
 **
-*F  ELM_WPOBJ(<wp>,<pos>) . . . . . . . . . . . . . element of a WP object
+*F  ELM_WPOBJ(<wp>, <pos>) . . . . . . . . . . . . . get entry of a WP object
 **
-**  'ELM_WPOBJ' return the <wp>-th element of the WP object <wp>.  <pos> must
+**  'ELM_WPOBJ' return the <pos>-th element of the WP object <wp>. <pos> must
 **  be a positive integer less than or equal  to the physical length of <wp>.
 **  If <wp> has no assigned element at position <pos>, 'ELM_WPOBJ' returns 0.
 **
 **  If the entry died at a recent garbage collection, it will return a Bag ID
```

Comment:
This comment seems to no longer be correct. While a garbage collection might occur almost immediately after a non-zero object has been returned, the reference from the C stack should keep it alive, so with this change it is safe to assume that any object you get back is real.

Reply:
Oops indeed. I could have sworn that I already removed this...?!? Thanks, will fix it ADAP